.github/workflows/build.yaml

```
@@ -47,8 +47,52 @@ jobs:
           name: InstallFiles
           path: InstallFiles/
 
-  macOS:
-    name: macOS
+
+  windows-debug:
+    name: windows-debug
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v1
+        id: cache-dxfiles
+        with:
+          path: utils/DXFiles
+          key: dxfiles-0001
+
+      - name: Download DirectX
+        if: steps.cache-dxfiles.outputs.cache-hit != 'true'
+        run: Invoke-WebRequest https://mirror.mtasa.com/bdata/DXFiles.zip -OutFile utils/DXFiles.zip
+        shell: powershell
+
+      - name: Extract DirectX
+        if: steps.cache-dxfiles.outputs.cache-hit != 'true'
+        run: Expand-Archive -Path utils/DXFiles.zip -DestinationPath utils/DXFiles
+        shell: powershell
+
+      - name: Install CEF
+        run: utils\premake5 install_cef
+
+      - name: Install data files
+        run: utils\premake5 install_data
+
+      - name: Run Build
+        run: win-build.bat Debug
+        shell: cmd
+        env:
+          DXSDK_DIR: './utils/DXFiles/'
+          CI: 'true'
+
+      - name: Create build artifacts
+        run: utils\premake5 compose_files
+
+      - uses: actions/upload-artifact@master
+        with:
+          name: InstallFiles
+          path: InstallFiles/
```

Might need to set a different upload filename for the debug build? (artifacts are workflow-wide)

```suggestion

      - name: Create build artifacts
        run: utils\premake5 compose_files

      - uses: actions/upload-artifact@master
        with:
          name: InstallFiles-debug
          path: InstallFiles/
```